ethportal-api/Cargo.toml

```diff
@@ -5,27 +5,23 @@ edition = "2021"
 description = "Definitions for various Ethereum Portal Network JSONRPC APIs"
 license = "GPL-3.0"
 repository = "https://github.com/ethereum/trin/tree/master/ethportal-api"
+readme = "README.md"
```

Comment:
Weird, I must have missed it when someone added GPL to this file. I don't think we've actually agreed on this as a group anywhere. The most important place for it to be would be the LICENSE file in the repo, which is missing.

Reply:
We discussed it on the sync, I think it was last thursday, when you were out. But happy to revisit this decision